composer.json

```diff
@@ -178,8 +178,8 @@
             "drupal/paragraphs": {
                 "https://www.drupal.org/project/paragraphs/issues/2901390": "https://www.drupal.org/files/issues/2020-06-25/paragraphs-2901390-51.patch"
             },
-            "drupal/views_taxonomy_term_name_depth": {
```

Comment:
Just to confirm, the `views_taxonomy_term_name_depth` went away right?

The simplesamlphp patch looks good.

Reply:
The module is still there... the patch is inherited by the dependencies.